tests/test_crud.py

```diff
@@ -314,17 +363,37 @@ def create_cgmlst_profile(self, sample, profile):
             'percent_called': percent_called,
             'profile': {}
         }
+
+
+        libraries = [{
+            'sample_id': sample.sample_id,
+            'sequencing_run_id':'TESTABC',
+            #'estimated_depth_coverage':40,
+            'most_abundant_species_name':'mtb',
+            "most_abundant_species_fraction_total_reads" : 90,
+            "estimated_genome_size_bp" : 12345,
+            "estimated_depth_coverage" : 40,
+            "total_bases" : 12345,
+            "average_base_quality" : 33,
+            "percent_bases_above_q30" : 95,
+            "percent_gc" : 55
+
+
+        }]
+
+        created_libraries = crud.create_libraries(self.session,libraries)
         for idx, locus_id in enumerate(locus_ids):
             profile_dict['profile'][locus_id] = profile[idx]
-        
-        created_cgmlst_profile = crud.create_cgmlst_allele_profile(self.session, profile_dict)
+
+        cgmlst_scheme = {'name':'Ridom cgMLST.org','version':'2.1','num_loci':2891} 
+        created_cgmlst_profile = crud.create_cgmlst_allele_profile(self.session,cgmlst_scheme, profile_dict,"TESTABC")
         json_serializable_cgmlst_profile = utils.row2dict(created_cgmlst_profile)
 
 
         note(json_serializable_cgmlst_profile)
-        assert(created_cgmlst_profile.sample_id == sample.id)
+        assert(created_cgmlst_profile.library_id == created_libraries[0].id)
           
 
         
 TestSampleCrudMachine = SampleCrudMachine.TestCase
-TestCgmlstAlleleProfileCrudMachine = CgmlstAlleleProfileCrudMachine.TestCase
+#TestCgmlstAlleleProfileCrudMachine = CgmlstAlleleProfileCrudMachine.TestCase
```

Comment:
Does this disable the test? If so, please leave a comment explaining why it's disabled.

Reply:
added